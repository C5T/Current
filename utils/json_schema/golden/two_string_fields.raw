{"Object":{"field_schema":[["a",{"String":{"values":{"b":1},"counters":[[1,"b"]],"instances":1,"nulls":0}}],["c",{"String":{"values":{"d":1},"counters":[[1,"d"]],"instances":1,"nulls":0}}]],"field_index":{"c":1,"a":0},"instances":1,"nulls":0}}