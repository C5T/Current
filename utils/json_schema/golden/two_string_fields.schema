// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Object) {
  CURRENT_FIELD(a, std::string);
  CURRENT_FIELD(c, std::string);
};

using Schema = Schema_Object;
