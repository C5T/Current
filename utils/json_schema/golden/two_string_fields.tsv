Field	Type	Set	Unset/Null	Values	Details
Schema	Object	1	0	2 fields	a, c
Schema.a	String	1	0	1 distinct value, "b":1
Schema.c	String	1	0	1 distinct value, "d":1
